src/bobril.ts

```diff
@@ -1503,6 +1503,43 @@ b = ((window: Window, document: Document): IBobrilStatic => {
         return r;
     }
 
+    function runMethod(ctx: IBobrilCtx, methodId: string, parms: any) {
```

Comment:
It would be nice if runMethod would return true if method was found (done == true).